AppDB/appscale/datastore/utils.py

```diff
@@ -203,7 +203,7 @@ def fetch_and_delete_entities(database, table, schema, first_key,
 
   last_key = first_key + '\0' + TERMINATING_STRING
 
-  logging.debug("Deleting application data in the range: {0} - {1}".
+  logger.debug("Deleting application data in the range: {0} - {1}".
```

Comment:
I think the logger defined in this module is imported by `appscale.datastore.scripts.datastore`, and the level is set there. For consistency, it might make sense to create the logger from the module name here and set the level for the `appscale.datastore` logger in the `appscale.datastore.scripts.datastore.main`.